flexport/latitude
# [LDS-26] Create Latitude Breadcrumb Component

## Description:
Adds a Latitude breadcrumb component. Includes a Storybook story, tests, and documentation. Partial migration of this PR: https://github.flexport.io/flexport/flexport/pull/53242

## Jira:
https://flexport.atlassian.net/browse/LDS-26

## Manual Testing Plan:
1. Run Storybook.
2. See Breadcrumb story, observe correct behavior when clicking links. You can reset the breadcrumbs to their original state using the "Reset Breadcrumbs" button in the 'Knobs' section. See screenshot section below for example.
3. Run Jest tests for the Breadcrumb component using the following: `yarn test __tests__/Breadcrumbs-test.js`
4 Confirm all tests pass successfully.

## Screenshots:
![Breadcrumbs-storybook](https://github.flexport.io/storage/user/1704/files/cede8600-ee8d-11e9-995f-1094eb87f459)
